App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Relay from 'react-relay';
+import { render } from 'react-dom';
+import { Modal } from 'xb-common-lib';
+import AppRouter from './AppRouter';
+
+import './node_modules/xb-common-lib/dist/xbCommonLib.css';
```

Comment:
No, we are not bundle css with js... We are to offer just css/js pure. xb-commo-lib has not webpack to bundle it and is using only babel to transpile.